tests/constraints/test_int_constraints.py

```diff
@@ -55,6 +55,7 @@ def test_handle_constrained_int_handles_lt(maximum: int) -> None:
     assert result < maximum
 
 
+@settings(suppress_health_check=(HealthCheck.filter_too_much,))
```

Comment:
what is this a symptom of?

Reply:
I'm not a 100% sure actually. I think it has something to do with the `assume(result == max_value)` but I'm not familiar enough with the value generators nor hypothesis enough to figure out exactly why that was needed.